pkg/controllers/deprovisioning/types.go

```diff
@@ -71,6 +72,11 @@ func NewCandidate(ctx context.Context, kubeClient client.Client, recorder events
 	if !node.Initialized() {
 		return nil, fmt.Errorf("state node isn't initialized")
 	}
+	// If the orchestration queue is already considering a candidate we want to deprovision, don't consider it a candidate.
+	if err := queue.CanAdd(node.Name()); err != nil {
+		recorder.Publish(deprovisioningevents.Blocked(node.Node, node.NodeClaim, "Already being deprovisioned")...)
```

Comment:
I've seen a decent amount of issues over the last year where a user wants to know why we're not taking action on a node. This is actually the same case of a non-consolidatable node. If a node is being deprovisioned for another reason, but maybe is waiting for a replacement to come up, this lets the user know on a per-node basis that we can't. 

Although, I could see a case for not including this, as a user might get a "terminating" and "blocking" event on the same node which could be confusing.

Reply:
Can't we just pop a status condition on the NodeClaim and call it a day? If the customer wants to know why, they can check their NodeClaim rather than trying to parse a complex stream of kube events.

Reply:
A status condition would require us to remove when the condition when we notice that it's no longer in a deprovisioning action, which may or may not be necessary depending on if it was successful due to another command.